ssl/d1_lib.c

```diff
@@ -959,3 +979,11 @@ size_t DTLS_get_data_mtu(const SSL *s)
 
     return mtu;
 }
+
+void DTLS_set_timer_cb(SSL *s, unsigned (*cb)(SSL *s, unsigned timeout_ms))
+{
+    if (!s || !s->d1)
+        return;
```

Comment:
No need to check this. We always assume that the main object is not NULL (otherwise its a programming error)...we don't have runtime checks for this. Similarly d1 can be assumed to be present.